Scripts/IncidentFields/IncidentFields.py

```diff
@@ -0,0 +1,77 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def get_field_by_long_name(fields, name):
+    for field in fields:
+        if field['name'] == name:
+            return field
+    return None
+
+
+def get_field_by_short_name(fields, name):
+    for field in fields:
+        if field['cliName'] == name:
+            return field
+    return None
+
+
+def get_short_name(value):
+    return re.sub(r'[^A-Za-z0-9]', '', value).lower()
+
+
+def main():
+    # get incident fields
+    res = demisto.executeCommand('demisto-api-get', {'uri': '/incidentfields'})
+    if is_error(res):
+        return_error(res[0]['Contents'])
+
+    fields = res[0]['Contents']['response']
+
+    # 'fields' contains non-incident fields, as well, so let's make a version containing only incident fields
+    incident_fields = [field for field in fields if field['id'].startswith('incident_')]
+
+    # get arguments
+    args = demisto.args()
+
+    custom_only = False
+    if 'custom' in args and argToBoolean(args['custom']) is True:
+        custom_only = True
```

Comment:
let's change the name of this argument to something more readable, like include_system_fields (if I understood what this stands for correctly).